temba/contacts/tasks.py

```diff
@@ -117,3 +123,97 @@ def check_elasticsearch_lag():
                 return False
 
     return True
+
+
+@nonoverlapping_task(track_started=True, name="block_deactivated_contacts_task")
+def block_deactivated_contacts_task():
+    email_subject = f"{timezone.now().strftime('%B %d, %Y')} - list of deactivated phone numbers."
+    email_text = """
+    Hi There!
+    We have prepared a list of contacts that were blocked because of deactivated phone number.
+    You can download it in the attached file.
+
+    Thanks,
+    The CommunityConnectLabs Team.
+    """
+
+    all_blocked_contacts = {}
+    for org in Org.objects.filter(is_active=True):
+        client = org.get_twilio_client()
+        if not client:
+            continue
+
+        try:
+            # get the link from twilio to download deactivated phone numbers
+            response = client.request("GET", "https://messaging.twilio.com/v1/Deactivations", {"Date": "2020-09-05"})
```

Comment:
Should this date be hard-coded?

Reply:
Deactivated phone numbers list is per Twilio account or is it global Twilio information?

Reply:
The deactivated phone numbers list is global Twilio information.

Reply:
As a date, I will set a previous day from the current date, because as I have tested API doesn't work with the current date.